sympy/core/evalf.py

```diff
@@ -284,10 +315,12 @@ def finalize_complex(re, im, prec):
     return re, im, re_acc, im_acc
 
 
-def chop_parts(value, prec):
+def chop_parts(value: TEMP_RESULT, prec: int) -> TEMP_RESULT:
     """
     Chop off tiny real or complex parts.
     """
+    if not isinstance(value, tuple):
+        return value
```

Comment:
When is this called with something that is not a tuple?

Reply:
same above